src/scripts/main.js

```diff
@@ -1,8 +1,25 @@
 'use strict';
 
-const people = require('./lib/people');
+const people = require('./lib/people.json');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(human => {
+  const { sex, born, died } = human;
+  const nameHuman = human.name;
+  const age = human.died - human.born;
+  const century = Math.ceil(human.died / 100);
+  const arr = [nameHuman, sex, born, died, age, century];
+  const row = document.createElement('tr');
+
+  for (let i = 0; i < arr.length; i++) {
+    const cell = document.createElement('th');
```

Comment:
please create `td` instead of `th`
```suggestion
    const cell = document.createElement('td');
```